collect_app/src/main/java/org/odk/collect/android/widgets/SelectOneSearchWidget.java

```diff
@@ -34,8 +36,10 @@
  */
 @SuppressLint("ViewConstructor")
 public class SelectOneSearchWidget extends AbstractSelectOneWidget implements OnCheckedChangeListener, AudioPlayListener {
+    Context mContext;
```

Comment:
Hey @akashdobaria you don't have to go through the hassle of creating a field to store the `Context` because underlying base class of this widget which is a `RelativeLayout` gets it via it's constructor. That's the reason for the call to  `super(context,prompt,false); `. 
So all you have to do is call `getContext()` and you are good to go :smile:

Reply:
Hi @jd-alexander ,
Thank you for your review. I'll make the changes and push it. :)

Reply:
Awesome :sunglasses: